be/issue-tracker/src/main/java/org/presents/issuetracker/global/error/statuscode/StatusCode.java

```diff
@@ -0,0 +1,11 @@
+package org.presents.issuetracker.global.error.statuscode;
+
+import org.springframework.http.HttpStatus;
+
+public interface StatusCode {
```

Comment:
이거 보면 `StatusCode` 를 구현하고 있는 구현체들은 항상 아래 필드를 가지고 있는데 abstract 로 바꾸는건 별로일까요?
Interface 도 좋은 선택같아 보이는데 한번 고민해보시면 좋을거 같아요.

```java
private final HttpStatus httpStatus;
	private final String message;
```
```suggestion
public abstract class StatusCode {
        protected HttpStatus httpStatus;
	protected String message;
	
	abstract String getName();

	abstract HttpStatus getHttpStatus();

	abstract String getMessage();
}
```